src/build/dotnet/jobs/nuget-package.job.yaml

```diff
@@ -1,26 +1,110 @@
+# Job to build a NuGet Package that will be hosted on an internal VSTS feed
```

Comment:
will this always be our internal nuget feed?

I think this and `nuget-package.job.yaml` should be named differently, i.e:
private/public-nuget-package, internal/external-nuget-package... etc - what do you think?

Reply:
does each of these have it's own corresponding deployment in deployment/nuget/jobs? If so, I think the names should be consistent

Reply:
The only difference with the build and the feed, is where it looks to find the current version number for the versioning task, don't think these need to be separate job files just for that, everything else would be duplicated.

Reply:
I think because the builds steps would be identical in these scenarios, there is no need to duplicate. There is only one small tweak based on the deployment feed which is for the Versioning task.